src/components/entity/ha-statistics-picker.ts

```diff
@@ -96,11 +110,10 @@ class HaStatisticsPicker extends LitElement {
       <div>
         <ha-statistic-picker
           .hass=${this.hass}
-          .includeStatisticsUnitOfMeasurement=${this
-            .includeStatisticsUnitOfMeasurement}
-          .includeUnitClass=${this.includeUnitClass}
+          .includeStatisticsUnitOfMeasurement=${includeStatisticsUnitCurrentLastPicker}
+          .includeUnitClass=${includeUnitClassCurrentLastPicker}
           .includeDeviceClass=${this.includeDeviceClass}
```

Comment:
a little surprised that includeDeviceClass is only on the last picker, and not on the other ones. Is that a bug?

Reply:
yes